torch/distributed/pipelining/schedules.py

```diff
@@ -2110,6 +2111,35 @@ def need_bubble(stage, op, microbatch, num_stages_global, seen_ops):
         return result
 
 
+class ScheduleInterleavedZeroBubble(ScheduleFlexibleInterleaved1F1B):
```

Comment:
nit: the paper does give names. we might want to use their name for the class name, or at very least mention which of their names this corresponds to in the comments for the class.